### PR TITLE
CTPPS: exceptions fix (port of #15084)

### PR DESCRIPTION
**Pull request by jan-kaspar**
- geometry: no (misleading) error is printed when no alignment corrections are loaded, LogVerbatim is used instead of printf and cout
- DQM: histograms created with fixed ranges to avoid problems when merging histograms from several module instances

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by dmitrijus** [comment]
Not sure if LogVerbatim is the right choice, shouldn't it be LogWarning?
I will let others comment.

**Comment by dmitrijus** [comment]
+1

**Comment by jan-kaspar** [comment]
@dmitrijus The messages in question have informative character - there is nothing to worry about in the content. Thus I've opted for `LogVerbatim` as non-formatted version of `LogInfo`.

File: Geometry/VeryForwardGeometryBuilder/plugins/TotemRPGeometryESModule.cc [hunk 1]
```diff
@@ -416,33 +416,34 @@ void TotemRPGeometryESModule::ApplyAlignments(const edm::ESHandle<DDCompactView>
...
-                << alignments->GetSensorMap().size() << " sensor alignments applied.\n";
-        }
-    } else {
-        if (verbosity)
-            cout << ">> TotemRPGeometryESModule::produceMeasuredDDCV > Measured geometry: No alignments applied.\n";
+  // load alignments
+  edm::ESHandle<RPAlignmentCorrectionsData> alignments;
+  try {
+    iRecord.getRecord<RPMeasuredAlignmentRecord>().get(alignments);
+  } catch (...) {}
```
Comment (ianna) [inline Geometry/VeryForwardGeometryBuilder/plugins/TotemRPGeometryESModule.cc]: @jan-kaspar - please, do not use catch(...) if indeed, you want to try this rather then asking for the handle validity. Please use proper exception handling: catch cms::Exception only, print a message, re-throw, etc...

**Comment by dmitrijus** [comment]
80x: https://github.com/cms-sw/cmssw/pull/15084 (proper reference I can click!)

**Comment by davidlange6** [comment]
@jan-kaspar - do you plan to update this pr?

**Comment by jan-kaspar** [comment]
@davidlange6 No. Regarding the "catch" comment from Ioanna, I've replied in the PR #15084:

> Although it may look like, I'm not adding the catch, it was in the code before. And it was there since in our workflow we allow not to insert any alignment corrections. In my patch I only remove the error message which was misleading since there is nothing wrong when no alignments are supplied.

Thus, unless you do want me to "fix" the catch also in this PR, I'm not intending to update it.

**Comment by ianna** [comment]
+1

**Pull request merged**